integrationtest/performance/orm/pom.xml

```diff
@@ -30,7 +30,7 @@
         <jbosshome>${project.build.directory}/${serverName}/</jbosshome>
 
         <!-- Relax the JVM restrictions for integration tests, when using Jigsaw -->
-        <additionalRuntimeArgLine>${additionalRuntimeArgLineForWildFlyTests}</additionalRuntimeArgLine>
+        <test.jvm.args>${test.wildfly.jvm.args}</test.jvm.args>
```

Comment:
> also doesn't really help to make it clear that this is not a version?

Well let's be reasonable, version properties start with `version.`, I think that's a pretty clear indication those are versions and other properties are not :) I don't see why people would start assuming a property is about a version just because it doesn't use camel case.

> why you consider this a cleanup

Simply because it's more consistent with other properties. For instance, other properties that include runtime args all end with `.args`: `surefire.system.args`, `failsafe.jacoco.args`, ... When properties with similar purposes have similar names, it's arguably easier to understand they are related.

I wouldn't have done it just for the sake of consistency, but since I was working on this part of the configuration, and had to understand how it worked before I made other changes, I thought I might as well update the configuration to reflect what I understood.

Now if you think those names are not clear enough, I can of course change them.

Reply:
don't worry, I don't care :)

I'm just suggesting you shouldn't care either, such changes just cause more work, risk of trouble, and _maybe_ they add some value but I doubt it.